src/server.h

```diff
@@ -103,7 +103,7 @@ class CHighPrecisionTimer : public QThread
 protected:
     virtual void run();
 
-    bool     bRun;
```

Comment:
Please revert this change.

Reply:
I did not change this.
If anything changed it, it was clang-format.